controllers/servingruntime_controller.go

```diff
@@ -75,6 +80,10 @@ type ServingRuntimeReconciler struct {
 	ClusterScope bool
 	// whether the controller is enabled to read and watch ClusterServingRuntimes
 	EnableCSRWatch bool
+	// whether the controller is enabled to read and watch secrets
+	EnableSecretWatch bool
+	// whether the controller is enabled to deploy pvc mounts based on predictors
+	DeployPVCForPredictor bool
```

Comment:
I was thinking that this would be a dynamic config parameter, added to the global configmap.